openc3/lib/openc3/utilities/authentication.rb

```diff
@@ -144,7 +144,7 @@ def _refresh_token(current_time)
 
     # Make the post request to keycloak
     def _make_request(headers, data)
-      uri = URI("#{@url}/auth/realms/openc3/protocol/openid-connect/token")
+      uri = URI("#{@url}/realms/openc3/protocol/openid-connect/token")
```

Comment:
I remember you talking about them removing auth from the base uri but is this backwards compatible? Do they support both?

Reply:
You have to modify the OPENC3_KEYCLOAK_URL environment variable with this change.